jobs/pxc-mysql/templates/db_init.erb

```diff
@@ -142,6 +142,9 @@ REVOKE ALL PRIVILEGES ON *.* FROM #{user['username']}@#{user['host']};
     if cfg['role'].nil? || cfg['role'].empty?
       raise("seeded_users property specifies an empty allowed 'role' for username #{username}")
     end
+    if cfg['plugin'].nil? || cfg['plugin'].empty?
+      cfg['plugin'] = 'mysql_native_password'
```

Comment:
Might be worth introducing a spec property to default this, rather than hardcoding.

In some cases, it might be convenient to configure an auth plugin on a per user basis, but in many cases a deployment would want to just set a server wide default and adjust the [my.cnf authentication-policy](https://github.com/cloudfoundry/pxc-release/blob/8e8d5bffc86432f7f7066391a83874c0d38180bf/jobs/pxc-mysql/templates/my.cnf.erb#L62) as well.

For the default, I would constrain to either `mysql_native_password` or `caching_sha2_password`.   That's probably fine to restrict for _any_ `plugin` option and avoids generating invalid SQL here.